addons/mail/static/src/components/chat_window/chat_window.scss

```diff
@@ -23,42 +17,26 @@
         }
     }
 
-    &.o-mobile {
-        position: fixed;
-    }
-
-    &.o-fullscreen {
-        height: 100%;
-        width: 100%;
-    }
 }
 
 .o_ChatWindow_channelInvitationForm {
     min-height: 450px; // allow flex shrink smaller than content (but not too small)
 }
 
 .o_ChatWindow_header {
-    flex: 0 0 auto;
-}
-
-.o_ChatWindow_newMessageForm {
-    padding: map-get($spacers, 1);
-    margin-top: map-get($spacers, 1);
-    display: flex;
-    align-items: center;
-}
+    flex-basis: auto;
+} 
 
 .o_ChatWindow_newMessageFormInput {
-    flex: 1 1 auto;
+    flex-basis: auto;
```

Comment:
Aren't all of those the default value, so could be removed from the file?

Reply:
They should indeed, although when I tested it without, it didn't render properly. I'll test it again to make sure.

Reply:
Tested and I can't seem to recreate my previous issue, I've removed them :)